addons/fieldmanual/CfgHints.hpp

```diff
@@ -0,0 +1,326 @@
+
+/*
+    Preferred Image Size = 256x256
+    Can Embed Images Into Description = 1024x512
+        - Shadow = '0'
+        - Size = '9'
+*/
+
+class CfgHints {
+    class ACE_FieldManual_Base {
+        arguments[] = {
+            // SELF INTERACT KEYBIND - %11
+            "
+                private _localized = 'Self Interact Key';
+                if !(isNil {['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind}) then {
+                    _localized = ['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind select 8 select 0 call CBA_fnc_localizeKey;
+                };
+                _localized
```

Comment:
I guess add trailing slash for multiline
```suggestion
            "\
                private _localized = 'Self Interact Key';\
                if !(isNil {['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind}) then {\
                    _localized = ['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind select 8 select 0 call CBA_fnc_localizeKey;\
                };\
                _localized\
            ",
```

Reply:
fixed